### PR TITLE
Updated create universal profile doc to use l16 & minor typo corrections

### PR DESCRIPTION
**Pull request by jxyang44**
Hey guys, I updated the "Create a Universal Profile" guide from L14 to L16 as I was going through the code.

I'm not sure if the docs were intentionally left to use L14 for universal.cloud functionality. Sorry if that's the case!

### CODE REVIEW

File: docs/guides/universal-profile/create-profile.md
Comment (Hugoo) [inline docs/guides/universal-profile/create-profile.md:109]: haha yeah, a singing method is probably a cool one but yeah... 😅🎶

**Review by Hugoo — APPROVED** [review]
Hey - thanks for the super cool PR @jxyang44 !
Indeed we are currently updating the docs to only mention L16 and not L14 anymore.
Thanks too for the typo updates!

Waiting for @fhildeb approval before merging.

**Review by fhildeb — APPROVED** [review]
Indeed, we originally left L14 here, so the deployed Universal Profile can be directly seen on universalprofile.cloud- but as Hugo mentioned, we plan on updating everything to use the latest network by default ;) Thanks for your PR!

**Pull request merged**
